### PR TITLE
Added DVS-Voltmeter Event Simulator

### PR DESCRIPTION
**Pull request by chahatdeep**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2508,7 +2508,9 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [Pro
 - <a name="Joubert21fins"></a>Joubert, C., Marcireau, A., Ralph, N., Jolley, A., van Schaik, A., Cohen, G.,  
 [Event Camera Simulator Improvements via Characterized Parameters](https://www.frontiersin.org/articles/10.3389/fnins.2021.702765/full)  
   Front. Neurosci., 2021, [IEBCS simulator](https://github.com/neuromorphicsystems/IEBCS)
-
+- <a name="Lin2022dvs"></a>Lin, S., Ma, Y., Guo, Z. and Wen, B.,
```
Comment (guillermogb) [inline README.md:2511]: key: Lin22eccv (if it does not collide with another entry)

File: README.md [hunk 2]
```diff
@@ -2508,7 +2508,9 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [Pro
...
+[DVS-Voltmeter: Stochastic process-based event simulator for dynamic vision sensors](https://www.ecva.net/papers/eccv_2022/papers_ECCV/papers/136670571.pdf)
+  European Conference on Computer Vision, 2022 Springer, [DVS-Voltmeter](https://github.com/Lynn0306/DVS-Voltmeter)
```
Comment (guillermogb) [inline README.md:2513]: - European Conf. Computer Vision (ECCV)   (see other entries)
- [DVS-Voltmeter]( ... )  -> [Code]( ... )

**Review by guillermogb — CHANGES_REQUESTED** [review]
thanks Chahat.

**Review by chahatdeep — COMMENTED** [review]
Welcome. :)

**Comment by guillermogb** [comment]
@chahatdeep one change is still pending. Could you take a look?

File: README.md (hunk 1 above)
Comment (guillermogb) [reply README.md:2511]: The line breaks are still missing.

image

Authors, title and publication in three separate lines (3 lines in total), please.

File: README.md (hunk 2 above)
Comment (guillermogb) [reply README.md:2513]: This change is also pending.

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Comment by likuui** [comment]
来信收到，谢谢！

**Comment by guillermogb** [comment]
The changes in this PR are included in #270 , so I am closing this one.

**Comment by guillermogb** [comment]
The changes in this PR are included in PR #270 , so I am closing this one.

**Pull request closed without merging**